vendor/Comment.php

```diff
@@ -0,0 +1,19 @@
+<?php
+include "User.php";
+class Comment
+{
+    private $user, $message;
+
+    public function __construct(User $user, String $message) {
```

Comment:
String должен быть с маленькой буквы и { должна быть с новой строки:
```suggestion
    public function __construct(User $user, string $message)
    {
```